charms/worker/k8s/charmcraft.yaml

```
@@ -375,9 +375,9 @@ config:
       default: true
       description: |
         Enable local storage provisioning. This will create a storage class
-        named "local-storage" that uses the hostPath provisioner. This is
-        useful for development and testing purposes. It is not recommended for
-        production use.
+        named "local-storage" that uses the hostPath provisioner. Local storage
```

it's not really a hostPath provisioner is it?
```suggestion
        named "local-storage" that uses the rawfile-localpv provisioner. Local storage
```